src/main/java/tech/ydb/io/r2dbc/YdbConnection.java

```diff
@@ -100,7 +101,12 @@ public ConnectionMetadata getMetadata() {
 
     @Override
     public IsolationLevel getTransactionIsolationLevel() {
-        return isolationLevel;
+        throw new UnsupportedOperationException(
```

Comment:
Maybe return SERIALIZABLE

Reply:
I think this is a bad practice. Ydb has different isolation levels from the standard ones. And it's better to talk about it right away using an exception in the method with the name of the required method, otherwise the user may get the impression that the method works, and it only works for the SERIALIZABLE isolation level.